clients/client/src/main/java/org/projectnessie/client/api/NessieApiV1.java

```diff
@@ -99,4 +99,7 @@
   MergeReferenceBuilder mergeRefIntoBranch();
 
   CommitMultipleOperationsBuilder commitMultipleOperations();
+
+  /** Retrieve diffs between two references. */
+  GetDiffsBuilder getDiffs();
```

Comment:
in the case of `Content` vs `Contents` it was more straight-forward, since `Content` would always only contain a single object. For the diffs however, we return a list of `Diff` objects, so it seemed more appropriate to me to use the plural form, but I don't have a strong opinion in this particular case, since `getDiff()` also makes sense to me.
@snazy thoughts?

Reply:
"diff" is probably the right term here. It's a single "diff" that's being asked for - the one diff contains a set/list of differences.

Reply:
updated to `GetDiffBuilder` and `getDiff`